src/metabase/sync_database/analyze.clj

```diff
@@ -259,3 +175,11 @@
             (log/info (u/format-color 'blue "%s Analyzed table '%s'." (u/emoji-progress-bar <> tables-count) table-name))))))
 
     (log/info (u/format-color 'blue "Analysis of %s database '%s' completed (%s)." (name driver) (:name database) (u/format-nanoseconds (- (System/nanoTime) start-time-ns))))))
+
+(defn analyze-database
+  "analyze all the tables in one database"
+  [db]
+  (analyze-data-shape-for-tables! (->> db
+                                       :id
```

Comment:
also, this lends me to believe that there shouldn't be a separate schedule for classification at this point. it should only run when triggered by a new fingerprint being available or when new classifiers are shipped (i.e. possibly a startup check if we ship new ones in the future).